phylogenetic/rules/config.smk

```diff
@@ -11,7 +65,31 @@ def conditional(option, argument):
     if argument is True: # must come before `isinstance(argument, int)` as bool is a subclass of int
         return [option]
     if isinstance(argument, list):
+        if len(argument)==0:
+            return "" # empty list interpreted as no command line args (empty string)
```

Comment:
Shouldn't this already be handled by the previous `not argument` condition?

https://github.com/nextstrain/ebola/blob/ddf65edb0e616a08549c211885568d69f2043975/phylogenetic/rules/config.smk#L63-L64

Reply:
Good catch! [Fixed](https://github.com/nextstrain/ebola/compare/ddf65edb0e616a08549c211885568d69f2043975..0db9dfdc2dae93da49be99bedd96e22deca5a81a)